templates/frontend/objects/article_details.tpl

```diff
@@ -120,6 +120,9 @@
 							{if $author->getData('orcid')}
 								<span class="orcid">
 									{$orcidIcon}
```

Comment:
Shouldn't this line be removed? If not the icon will likely appear twice...

Reply:
@mhvezina  thanks for  your review. 

Actullay, no.    AcessToken is only checked for displaying the orcidLogo. Here is the current  stable-3_3_0  branch.

https://github.com/pkp/ojs/blob/stable-3_3_0/templates/frontend/objects/article_details.tpl#L120

Reply:
I meant this line : https://github.com/pkp/ojs/pull/3865/files#diff-959b67b46d8905ed019270b42e656fe1286d6747716787b4681b5de6710c1eaeL122  But indeed now I see the line has been removed in https://github.com/pkp/ojs/blob/stable-3_3_0/templates/frontend/objects/article_details.tpl#LL121C23-L121C23 Thanks.